openproblems/tasks/_batch_integration/batch_integration_feature/metrics/hvg_conservation.py

```diff
@@ -27,7 +27,15 @@
 def hvg_conservation(adata):
     from scib.metrics import hvg_overlap
 
+    hvg = 2000
+
+    if adata.uns["n_genes_pre"] < hvg and adata.uns["n_genes_pre"] is not adata.n_vars:
+        return -1
+    if adata.n_vars < hvg:
+        return -1
```

Comment:
Can't we just run the metric as desired here? The intersection between the two could be 10 (or 0) genes, so why block on the number of genes before intersection?

Reply:
this is so you can't just pass a small number of genes, the first one is for the test data with 500 genes. @LuckyMD